bot/extensions/fun/weather.py

```diff
@@ -0,0 +1,129 @@
+from geopy.geocoders import Nominatim
+from timezonefinder import TimezoneFinder
+import pytz
+from pytz import country_timezones, timezone
+from datetime import datetime
+from discord.ext.commands import Cog, command
+import requests
+from discord import Embed
+import time
+import string
+
+from discord.utils import parse_time
+
+class WeatherCog(Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @command(name='weather', help='Show weather information in your city', usage="{city}")
+    async def my_command(self, ctx, *city_input):
+        # get all argument inputed by the user 
+        # on discord and join them together 
+        # (handle double-words/argument cities)
+        city = " "
```

Comment:
Creating a variable for that is a bit overkill. Just put the `" "` on the line bellow.